### ISSUE TITLE
Incorrect ordering of arguments

### ISSUE THREAD

**Issue by ishan00** [kamalkraj/BERT-NER#44]
Hey,
I was looking at the code and noticed that there is an error in the ordering of parameters given to BERT in line 486 of `run_ner.py` file

BertForTokenClassification (https://huggingface.co/pytorch-transformers/model_doc/bert.html#pytorch_transformers.BertForTokenClassification) expects the order to be `input_ids`, `attention_mask`, `token_type_ids`, `position_ids`, `head_mask`, `labels` whereas in the code as one can see below you have passed `segment_ids` (== `token_type_ids`) before `input_mask` (== `attention_mask`). Also `label_ids` (== `labels`) and `l_mask` (== `position_ids`) should also be switched.

```
Line 485-486
input_ids, input_mask, segment_ids, label_ids, valid_ids,l_mask = batch
loss = model(input_ids, segment_ids, input_mask, label_ids,valid_ids,l_mask)
```

The above also in line 555

Correct me if I've understood incorrectly

**Comment by kamalkraj** [comment]
Hi @ishan00,
I have different `forward` function https://github.com/kamalkraj/BERT-NER/blob/6286a475b3f85ad595f37b27d1e0d2a31e6edf70/run_ner.py#L32 .

I have inherited `BertForTokenClassification` from `pytorch_transformers` and redeifned  the `forward` function, where the ordering of arguments are different compared to  `BertForTokenClassification`

**Comment by ishan00** [comment]
Thanks for clarification. My mistake
